src/api/libcellml/importer.h

```diff
@@ -67,15 +67,15 @@ class LIBCELLML_EXPORT Importer: public Logger
      * @brief Resolve all imports in the @p model.
      *
      * Resolve all @c Component and @c Units imports by loading the models
-     * from local disk through relative URLs.  The @p baseFile is used to determine
-     * the full path to the source model relative to this one.
+     * from local disk through relative URLs.  The @p basePath defines
+     * the full path to the directory that import URLs are relative to.
      *
      * @param model The @c Model whose imports need resolution.
-     * @param baseFile The @c std::string location on local disk of the source @c Model.
+     * @param basePath The (effective) directory on local disk of the source @c Model.
```

Comment:
I don't know what `(effective)` means here. Perhaps clearer to have something more like: The full path used to resolve relative import URLs. ?

Reply:
49c3f4e